log_config.py

```diff
@@ -0,0 +1,69 @@
+import json
+from datetime import datetime
+from logging import Formatter, config, getLogger
+
+
+class JsonFormatter(Formatter):
+    def format(self, record):
+        log_record = {
+            "timestamp": datetime.utcnow().isoformat(),
```

Comment:
issue: `utcnow()` method is deprecated. import `UTC` from datetime and change to this:

```suggestion
            "timestamp": datetime.now(UTC).isoformat(),
```